maxim/logger/pydantic_ai/client.py

```
@@ -0,0 +1,737 @@
+                if display_name_fn:
+                    final_op_name = display_name_fn(processed_inputs)
+
+            except Exception as e:
+                scribe().debug(f"[MaximSDK] Failed to process inputs for {base_op_name}: {e}")
+                processed_inputs = {"self": self, "args": args, "kwargs": kwargs}
+
+            trace = None
+            span = None
+            generation = None
+            tool_call = None
+            trace_token = None
```

Ruff (0.12.2)

Local variable `trace_token` is assigned to but never used

Remove assignment to unused variable `trace_token`

(F841)